package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cimpress-fulfiller-identity",
-  "version": "0.1.4",
+  "version": "0.3.0",
```

Comment:
Why straight to version 3?

Reply:
I was playing with npm comands to update the version programatically. The correct version is 0.1.5